### PR TITLE
Add install scripts and installation instructions

### PR DESCRIPTION
**Pull request by runphilrun**
- add bash scripts for mac/linux installations
- add batch scripts for windows
- rearrange script locations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# RIT-SPEX/latex-utils
```
Comment (EvanMPutnam) [inline README.md:1]: Maybe add a section for pre-requisite software.  At this point it should only be Python 3.X and optional stuff for different types of compilers.  However, install scripts should alleviate some of that pain.

File: Install_Scripts/install_mactex.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "Installing MikTeX with Homebrew"
+brew cask install mactex
```
Comment (EvanMPutnam) [inline Install_Scripts/install_mactex.sh]: Brew is not installed by default on Mac machines.  It comes bundled with XCode command line tools.  Might want to do something similar to what we do for the windows install or make a note of it in pre-reqs if that is something we want to live with.

File: Portable_PDD/Compile.bat [hunk 3]
```diff
@@ -5,11 +5,14 @@ set DOCUMENT_PATH="markdown_template.md"
 REM This is where you specify the final name of your document.  Spaces will be replaced with underscores.
 set FINAL_DOC_NAME="50_SAT"
 
+REM This is the where you specify the template of the .tex file.
+set PDD_TEMPLATE="spex_template.tex"
+
 REM You can specify the type of compiler script.  It expects an input of the path of the .tex document.
-set COMPILE_SCRIPT="Compile_Scripts\\DockerCompiler.bat"
+set COMPILE_SCRIPT="Compile_Scripts\\MikTexCompiler.bat"
```
Comment (EvanMPutnam) [inline Portable_PDD/Compile.bat]: When you merged you relocated the two folders so this path needs to change to ..\Compile_Scripts

File: Install_Scripts/install_mactex.sh (hunk 2 above)
Comment (runphilrun) [reply Install_Scripts/install_mactex.sh]: I replaced this script with putting this command in the readme text. From my experience it is completely fine to assume a Mac developer has Homebrew installed.

**Pull request merged**